### ISSUE TITLE
User management results in a 500 error

### ISSUE THREAD

**Issue by pallaswept** [openSUSE/open-build-service#14595]
This was closed but never resolved. 

https://github.com/openSUSE/open-build-service/issues/10500

**Comment by hennevogel** [comment]
Are you able to reproduce it?

**Comment by pallaswept** [comment]
Of course, that's how I found that issue, I searched for the 500 error I received when trying to recover a deleted user (actually, I was trying to delete it entirely but if I can't delete and re-create it as I intended then I need to re-enable it).

Also, the in-built Admin account gets a 
`Server returned an error: HTTP Error 401: Unauthorized`
When applying the workaround kindly provided by the OP in that issue. So, I had to use the web interface to create a new dummy user, give them admin rights, and use that account to perform the workaround. Now I have a dummy admin account I can never delete.

**Comment by hennevogel** [comment]
So can you please explain how we can reproduce this? Basically answer the questions of the issue template....

How to Reproduce
===============
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error


Further Information
===============
* A link to an OBS instance showing the issue
* Exact server version (in case a private instance is used)

**Comment by pallaswept** [comment]
Briefly:
Login to your OBS instance webpage as an admin user
Go to configuration
Go to Manage Users (These three steps can be replaced with visiting the URL https://obs.instance.url/users)
Click the icon to edit the deleted user (The blue notepad with pen icon)
Select a state other than deleted (Any of the radio buttons provided)
Click 'Update' to apply your change
Get a 500 error page

**Comment by hennevogel** [comment]
And the second part of the question....?

**Comment by pallaswept** [comment]
https://download.opensuse.org/repositories/OBS:/Server:/2.10/images/obs-server.x86_64-qcow2.qcow2

Just try it
